### PR TITLE
output: remove a couple more "(s)" left

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Continuation of https://github.com/iterative/dvc/pull/2281#issuecomment-513053199

* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

Doc update: https://github.com/iterative/dvc.org/pull/490/commits/3dc54890cee4b9fec6e97134a46a47693dba14c1

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Note the tests are failing due to a problem in the upstream master branch:

```console
________ ERROR collecting tests/func/test_s3.py  _________
ImportError while importing test module '/Users/drbidibombom/dvc/tests/func/test_s3.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/func/test_s3.py:3: in <module>
    from moto import mock_s3
E   ModuleNotFoundError: No module named 'moto'
```

**Comment by efiop** [comment]
@jorgeorpinel Try `pip install git+https://github.com/efiop/moto.git@move-env-mocking`, This is a temporary moto version that we are using. Sorry for the inconvenience, I'll add a fixup ASAP.

**Comment by jorgeorpinel** [comment]
Np I'm just glad the Travil builds pass )

File: dvc/command/update.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    UPDATE_HELP = "Update dependencies and reproduce specified DVC-files."
+    UPDATE_HELP = "Update data artifacts imported from other DVC repositories."
```
Comment (jorgeorpinel) [inline dvc/command/update.py]: Please note this change is due to https://github.com/iterative/dvc.org/pull/490#discussion_r305584059. Feel free to resolve this comment after reading.

**Pull request merged**